src/steps/generic.rs

```diff
@@ -1344,3 +1345,34 @@ pub fn run_zigup(ctx: &ExecutionContext) -> Result<()> {
 
     Ok(())
 }
+
+pub fn run_jetbrains_toolbox(_ctx: &ExecutionContext) -> Result<()> {
+    let installation = find_jetbrains_toolbox();
+    match installation {
+        Err(FindError::NotFound) => {
+            // Skip
+            Err(SkipStep(format!("{}", t!("No JetBrains Toolbox installation found"))).into())
```

Comment:
> the t! strings don't actually have localization right now

That's because the message wrapped by `t!` has not been added to https://github.com/topgrade-rs/topgrade/blob/main/locales/app.yml:

```yml
"No JetBrains Toolbox installation found":
  en: "No JetBrains Toolbox installation found"
  es: "No se encontró ninguna instalación de JetBrains Toolbox"
  zh_TW: "找不到 JetBrains Toolbox 安裝"
```